docs/surveys/salary-survey/2022.rst

```diff
@@ -0,0 +1,2208 @@
+   - Negative
+   - Neutral
+   - Positive
+   - Very positive
+
+.. raw:: html
+
+   </details>
+
+Affected by COVID-19
+--------------------
+
+53.9% of respondents reported that their workplace had changed due to COVID-19 - down from 73.9% in 2021, and 80% in 2020. This decrease reflects the number of people who have started new jobs since the pandemic started.
+
+As in previous years, the bulk of the changes were moving from on-site to remote or partially remote. 
+
+.. table:: Work location - pre-COVID-19 (2022)
+   :width: 70%
+   :name: tbl-2022-work-location-pre-covid19
+
+   +-------------------------+-----+--------+
+   | Work location           | No. | %      |
+   +=========================+=====+========+
+   | On-site (required)      | 138 | 48.76% |
+   +-------------------------+-----+--------+
+   | Partial                 | 57  | 20.14% |
+   +-------------------------+-----+--------+
+   | On-site (not required)  | 34  | 12.01% |
+   +-------------------------+-----+--------+
+   | Remote (not required)   | 32  | 11.31% |
+   +-------------------------+-----+--------+
+   | Remote (required)       | 22  | 7.77%  |
+   +-------------------------+-----+--------+
+
+.. table:: Work location - current (2022)
+   :width: 70%
+   :name: tbl-2022-work-location-current
+
+   +------------------------+-----+--------+
+   | Work location          | No. | %      |
+   +========================+=====+========+
+   | Remote (not required)  | 130 | 45.94% |
+   +------------------------+-----+--------+
+   | Partial                | 97  | 34.28% |
+   +------------------------+-----+--------+
+   | Remote (required)      | 43  | 15.19% |
+   +------------------------+-----+--------+
+   | On-site (required)     | 8   | 2.83%  |
+   +------------------------+-----+--------+
+   | On-site (not required) | 5   | 1.77%  |
+   +------------------------+-----+--------+
+
+.. raw:: html
+
+   <figure>
+      <object role="img" aria-label="Work location (affected by COVID-19) (2022)" aria-describedby="figure-work-location-affected-desc" type="image/svg+xml" data="/_images/2022-work-location-affected.svg">
+         <p id="figure-work-location-affected-desc">Two donut charts showing work location, pre-COVID-19 and currently, for respondents who indicated that their workplace had been affected by the pandemic.</p>
+      </object> 
+      <figcaption>Figure: Work location (affected by COVID-19) (2022)</figcaption>
+   </figure>
+
+.. figure:: images/2022/2022-work-location-affected.svg
+   :class: hide
+
+
+47.3% of respondents in this category reported that their location change was now permanent. 
+
+Not affected by COVID-19
+------------------------
+
+46.1% of respondents indicated that their current work location was unaffected by COVID-19. Of these, the majority worked remotely - 41.3% were required to be remote, and 36.8% were remote by choice. 13.6% were partially remote, and only 8.2% worked onsite (4.5% by requirement and 3.7% by choice).
```

Comment:
feedback: we use onsite here, but further down, we use on-site. I prefer onsite, but whatever we use, we should be consistent.

Reply:
eagle eyes! I intended to standardize with on-site but missed 5 in total, have fixed :)